doc/doxygen/references.bib

```diff
@@ -771,6 +771,31 @@ @article{Riviere1999
   title = {Improved energy estimates for interior penalty, constrained and discontinuous {G}alerkin methods for elliptic problems. {P}art {I}}.
 }
 
+
+% ------------------------------------
+% Step 74
+% ------------------------------------
+
+@book{di2011mathematical,
+  title={Mathematical aspects of discontinuous Galerkin methods},
+  author={Di Pietro and Daniele Antonio and Ern, Alexandre},
```

Comment:
What's the first name of Di Pietro? I think that both of these are actually part of his last name.

Reply:
``Di Pietro, Danielle Antonio and Ern, Alexandre``

Reply:
https://www.springer.com/gp/book/9783642229794